masterlist.yaml

```diff
@@ -2139,6 +2252,9 @@ plugins:
     after:
       - 'SkyrimIsWindy.esp'
       - 'SRG Enhanced Trees Activator.esp'
+  - name: 'TerrainLodRedone.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/9135/' ]
+    group: *earlyLoadersGroup
```

Comment:
Set group to early loaders. Must load high to prevent it overwritting mods which add their own LOD such as Open Cities.
[Source](https://forums.nexusmods.com/index.php?/topic/5486077-lodterrain-lod-redone/?p=52022082 ) & Mod [description page]( https://www.nexusmods.com/skyrimspecialedition/mods/9135/ )